easybuild/easyconfigs/r/RASPA2/RASPA2-2.0.47-foss-2022a.eb

```
@@ -0,0 +1,34 @@
+easyblock = 'ConfigureMake'
+
+name = 'RASPA2'
+version = '2.0.47'
+
+homepage = 'https://iraspa.org/raspa/'
+description = """RASPA is a software package for simulating adsorption and diffusion of
+molecules in flexible nanoporous materials."""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+source_urls = ['https://github.com/iRASPA/RASPA2/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['1a14524f95ab86e7a2910b53641262ec98ec23f794f1c3665982c32efc1997ed']
+
+builddependencies = [
+    ('Autotools', '20220317')
+]
+
+preconfigopts = 'rm -rf autom4te.cache && '
+preconfigopts += 'mkdir m4 && '
+preconfigopts += 'aclocal && '
+preconfigopts += 'autoreconf -i && '
+preconfigopts += 'automake --add-missing && '
+preconfigopts += 'autoconf && '
```

I think this can just be simplified to 
```suggestion
preconfigopts += 'autoreconf -if && ''
```

---

in fact, I think the whole `preconfigopts` could be reduced to that. `-i` installs missing files, and `-f` considers all files obsolete. 
> `autoreconf` will run `autoconf`, `autoheader`, `aclocal`, `automake`, `libtoolize`, and `autopoint` (when appropriate) repeatedly

---

@levlafayette any thoughts on the suggestions from @jfgrimm?